src/main/java/it/lucchesi/tools/DateTools/DateTools.java

```diff
@@ -26,6 +26,8 @@ public static long weekdays(LocalDate start, LocalDate end) {
 		Preconditions.checkArgument(Preconditions.checkNotNull(start)
 				.compareTo(Preconditions.checkNotNull(end)) <= 0);
 		
+		if(start.equals(end)) return 1;
```

Comment:
Yes, you are right. I got confused since it was passing the tests. Maybe it would be better to add a test where start and end are the same date but they are not weekdays.